app/src/main/java/com/babylon/wallet/android/domain/model/factors/FactorSourceCard.kt

```diff
@@ -0,0 +1,27 @@
+package com.babylon.wallet.android.domain.model.factors
```

Comment:
Should such model be kept in a `ui.model` package?

Reply:
And maybe start putting such models in that package now that we use them more?

Reply:
Good point